cmd/setupCi.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/kubefirst/kubefirst/internal/ciTools"
+	"github.com/kubefirst/kubefirst/internal/flagset"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// setupCiCmd represents the setupCi command
+var setupCiCmd = &cobra.Command{
+	Use:   "setup-ci",
+	Short: "A brief description of your command",
+	Long:  `TBD`,
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
As a reminder, to rename it to `add-ci`, so we can have an "anti" command to remove CI.